library/core/resource_loader/src/main/java/org/quiltmc/qsl/resource/loader/api/ResourceLoader.java

```diff
@@ -52,36 +53,65 @@ static ResourceLoader get(ResourceType type) {
 	/**
 	 * Registers a built-in resource pack.
 	 * <p>
-	 * A built-in resource pack is an extra resource pack provided by your mod which is not always active, it's similar to the "Programmer Art" resource pack.
+	 * A built-in resource pack is an extra resource pack provided by your mod which is not always active,
+	 * similar to the "Programmer Art" resource pack.
 	 * <p>
-	 * Why and when to use it?
 	 * A built-in resource pack should be used to provide extra assets/data that should be optional with your mod but still directly provided by it.
 	 * For example, it could provide textures of your mod in another resolution, or could allow providing different styles of your assets.
 	 * <p>
 	 * The path in which the resource pack is located is in the mod JAR file under the {@code "resourcepacks/<id path>"} directory.
 	 * {@code id path} being the path specified in the identifier of this built-in resource pack.
 	 * <p>
-	 * This method will fetch automatically the {@linkplain ModContainer} based on the namespace provided in {@code id}.
+	 * This method will automatically fetch the {@linkplain ModContainer} based on the namespace provided in {@code id}.
 	 *
-	 * @param id             the identifier of the resource pack, its namespace must be the same as the mod id
+	 * @param id             the identifier of the resource pack; its namespace must be the same as the mod id
 	 * @param activationType the activation type of the resource pack
 	 * @return {@code true} if successfully registered the resource pack, otherwise {@code false}
```

Comment:
If you're also going to fix some of the javadoc, then
```suggestion
	 * @return {@code true} if the resource pack was successfully registered, otherwise {@code false}
```